### PR TITLE
Migrate Progres

### PR DESCRIPTION
**Pull request by elias-ba**
## Summary

This PR migrates the language-progres adaptor inside this adaptors monorepo.

## Issues

#50

## Review Checklist

Before merging, the reviewer should check the following items:

- [x] Does the PR do what it claims to do?
- [x] If this is a new adaptor, has the migration tool been run and the
      migration guide followed?
- [x] Are there any unit tests? Should there be?
- [x] Is there a changeset associated with this PR? Should there be? Note that
      dev only changes don't need a changeset.

### CODE REVIEW

**Comment by mtuchi** [comment]
@eliaswalyba i am not seeing `ast.json` did you run `pnpm build` ?

**Comment by elias-ba** [comment]
> @eliaswalyba i am not seeing `ast.json` did you run `pnpm build` ?

@mtuchi nope. Is that step in the migration guide ? I used to stop at adding a changeset.

**Comment by mtuchi** [comment]
> > @eliaswalyba i am not seeing `ast.json` did you run `pnpm build` ?
> 
> @mtuchi nope. Is that step in the migration guide ? I used to stop at adding a changeset. 
@elias-ba 
I think it's missing in migration guideline but should make sure you run `pnpm build` in that adaptor that your migrating

**Comment by elias-ba** [comment]
> > > @eliaswalyba i am not seeing `ast.json` did you run `pnpm build` ?
> > 
> > 
> > @mtuchi nope. Is that step in the migration guide ? I used to stop at adding a changeset.
> > @elias-ba
> > I think it's missing in migration guideline but should make sure you run `pnpm build` in that adaptor that your migrating

Okay @mtuchi I can do that. Should I also add that to the migration guide ?

**Comment by mtuchi** [comment]
> > > > @eliaswalyba i am not seeing `ast.json` did you run `pnpm build` ?
> > > 
> > > 
> > > @mtuchi nope. Is that step in the migration guide ? I used to stop at adding a changeset.
> > > @elias-ba
> > > I think it's missing in migration guideline but should make sure you run `pnpm build` in that adaptor that your migrating
> 
> Okay @mtuchi I can do that. Should I also add that to the migration guide ?

@elias-ba yes add `in migration guideline, right after `pnpm install`

**Comment by elias-ba** [comment]
@mtuchi now ran `pnpm build` and added ast.json file to PR. Thanks

File: packages/progres/src/Adaptor.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import {
+  execute as commonExecute,
+  composeNextState,
+  expandReferences,
+  http,
+} from '@openfn/language-common';
+
+/**
+ * Execute a sequence of operations.
+ * Wraps `language-common/execute`, and prepends initial state for http.
```
Comment (mtuchi) [inline packages/progres/src/Adaptor.js:11]: It should say Wraps `@openfn/language-common/execute`

**Review by taylordowns2000 — APPROVED** [review]

**Pull request merged**